app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/dialogs/SendMessageDialogFragment.java

```diff
@@ -2,67 +2,43 @@
 
 import android.app.Activity;
 import android.app.Fragment;
-import android.content.Context;
-import android.view.View;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.view.LayoutInflater;
 import android.widget.EditText;
-import android.widget.Toast;
 
 import com.teamagam.gimelgimel.R;
 import com.teamagam.gimelgimel.app.GGApplication;
 import com.teamagam.gimelgimel.app.injectors.components.DaggerMessagesComponent;
 import com.teamagam.gimelgimel.app.injectors.modules.MessageModule;
 import com.teamagam.gimelgimel.app.view.fragments.dialogs.base.BaseDialogFragment;
+import com.teamagam.gimelgimel.app.viewModels.SendMessageDialogFragmentViewModel;
+import com.teamagam.gimelgimel.databinding.DialogSendMessageBinding;
 import com.teamagam.gimelgimel.presentation.presenters.SendMessagePresenter;
-
 import javax.inject.Inject;
-
 import butterknife.BindView;
 
 /**
- * Fragment that send
- * todo: complete
+ * Fragment that send a textual message
  */
 public class SendMessageDialogFragment extends BaseDialogFragment {
 
-    @BindView(R.id.dialog_send_geo_message_edit_text)
+    @BindView(R.id.dialog_send_text_message_edit_text)
     EditText mSendMessageEditText;
 
-    //    private GGMessageSender mMessageSender;
     @Inject
     SendMessagePresenter sendMessagePresenter;
 
-    @Override
-    public void onAttach(Activity activity) {
-        super.onAttach(activity);
-//        mMessageSender = ((GGApplication) getActivity().getApplicationContext()).getMessageSender();
-        DaggerMessagesComponent.builder()
-                .applicationComponent(((GGApplication) getActivity().getApplication()).getApplicationComponent())
-                .messageModule(new MessageModule())
-                .build()
-                .inject(this);
-    }
-
-    @Override
-    protected void onCreateDialogLayout(View dialogView) {
-    }
+    private DialogSendMessageBinding mBinding;
+    private SendMessageDialogFragmentViewModel mSendMessageViewModel;
 
     @Override
-    protected void onPositiveClick() {
-        String userMessage = mSendMessageEditText.getText().toString();
-
-        if (!isInputValid(userMessage)) {
-            showError();
-            sLogger.userInteraction("Clicked OK - invalid input");
-        } else {
-            sLogger.userInteraction("Clicked OK");
-            sendMessagePresenter.setView(new handlePresenterView(getActivity()
-                    .getApplicationContext()));
-            sendMessagePresenter.sendMessage(userMessage);
-
-//            mMessageSender.sendTextMessageAsync(userMessage);
-            dismiss();
-        }
-
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mBinding = DataBindingUtil.inflate(LayoutInflater.from(getActivity()), R.layout.dialog_send_message, null, false);
```

Comment:
What's the difference between DataBindingUtil.inflate() and DataBindingUtils.setContentView()?